web/app/__init__.py

```diff
@@ -31,6 +32,7 @@ def get_app(config):
     app.logger.addHandler(stream_handler)
 
     db.init_app(app)
+    Migrate(app, db)
```

Comment:
It registers the app with the Migrate flask extension. That way, the Migrate knows which app + db models to look at when `python manage.py db {init, upgrade, migrate, etc.}` is called.